tests/service/test_service_disable.yml

```diff
@@ -66,3 +75,6 @@
     ipaservice:
       ipaadmin_password: SomeADMINpassword
       name: "mysvc1/{{ ansible_fqdn }}"
+
+  - name: Destroy Kerberos tickets.
+    shell: kdestroy -A -q
```

Comment:
Instead of destroying all tickets isn't better to give a cachename in the init and again in the destroy?